docs/browser.md

```diff
@@ -20,16 +20,34 @@ approach comes with a lot of advantages, but also has some drawbacks:
 
 - Longer warm-up time
 - Increased risk of trade collisions
-- Consumes more end-user resources 
+- Consumes more end-user resources
 
-## Installing
+## @0x/mesh-browser
 
 For your convenience, we've published an NPM package called `@0x/mesh-browser`
-which includes the WebAssembly bytecode and a lightweight wrapper around it. You
-install this package in exactly the same way as any other NPM package and using
-it feels exactly like using a native TypeScript/JavaScript library.
+which includes the WebAssembly bytecode and a lightweight wrapper around it. To
+find more information on optimizing the startup time of using Mesh in the browser,
+refer to the next section on the `@0x/mesh-browser-lite` package. This package is
+used exactly the same way as any other NPM package and using it feels exactly
+like using a native TypeScript/JavaScript library.
```

Comment:
IMO this still reads like `@0x/mesh-browser-lite` is the one that "feels exactly like using a native TypeScript/JavaScript library". If anything it's the opposite. `@0x/mesh-browser` is more like a typical TypeScript/JavaScript library since it bundles everything together for you (at the cost of increasing load times) and does not require you to separately load the Wasm bytecode.

Can we just rearrange the sentences to make it more clear?

```suggestion
which includes the WebAssembly bytecode and a lightweight wrapper around it. This
package is used exactly the same way as any other NPM package and using it feels exactly
like using a native TypeScript/JavaScript library. To find more information on optimizing the
startup time of using Mesh in the browser, refer to the next section on the
`@0x/mesh-browser-lite` package.
```